doc/operator-capabilities.md

```diff
@@ -0,0 +1,193 @@
+# Operator Capability Levels
```

Comment:
```suggestion
# Operator Capability Levels with OLM
```

Reply:
Operator Capability levels having no relationship to OLM. OLM is not aware of it.